src/lib/model/PartiesModel.js

```diff
@@ -9,244 +9,56 @@
     const tokens = ['parties', type, id, subId];
     return tokens.map(x => `${x}`).join('-');
 }
 
 /**
- * 
- * @param {object} cache
-*  @param {string} type     - the party type
- * @param {string} id       - the party id
- * @param {string} subId    - the party subId, could be undefined!
- * @param {object} message  - the message used to trigger deferred job
- * @returns {Promise} - the promise which resolves when deferred job is invoked
- */
-function triggerDeferredJob({ cache, type, id, subId, message }) {
-    const cn = channelName(type, id, subId);
-    return deferredJob(cache, cn).trigger(message);
-}
-
-/**
- * @name generateKey
- * @description generates the cache key used to store state machine
- * @param {string} type     - the party type
- * @param {string} id       - the party id
- * @param {string} [subId]  - the optional party subId
- * @returns {string} - the cache key
- */
-function generateKey(type, id, subId) {
-    return `key-${channelName(type, id, subId)}`;
-}
-
-
-/**
- * @name injectHandlersContext
- * @description injects the config into state machine data, so it will be accessible to on transition notification handlers via `this.handlersContext`
- * @param {Object} config   - config to be injected into state machine data
- * @returns {Object}        - the altered specStateMachine
+ * @name requestAction
+ * @description invokes the call to switch
+ * @param {object} requests - MojaloopRequests instance
+ * @param {array} args - the arguments passed as object to `run` method
  */
-function injectHandlersContext(config) {
-    return { 
-        ...specStateMachine,
-        data: {
-            handlersContext: {
-                config: { ...config }, // injects config property
-                requests:  new MojaloopRequests({
-                    logger: config.logger,
-                    peerEndpoint: config.peerEndpoint,
-                    alsEndpoint: config.alsEndpoint,
-                    dfspId: config.dfspId,
-                    tls: config.tls,
-                    jwsSign: config.jwsSign,
-                    jwsSignPutParties: config.jwsSignPutParties,
-                    jwsSigningKey: config.jwsSigningKey,
-                    wso2Auth: config.wso2Auth
-                })
-            }
-        }
-    };
+function requestAction(requests, { type, id, subId }) {
+    return requests.getParties(type, id, subId);
 }
 
-
 /**
- * @name create
- * @description creates a new instance of state machine specified in specStateMachine ^
- * @param {Object} data     - payload data
- * @param {String} key      - the cache key where state machine will store the payload data after each transition
- * @param {Object} config   - the additional configuration for transition handlers
+ * @name argsValidationMethod
+ * @description makes validation of args object, invoked in `run, triggerDeferredJob, generateKey` methods to ensure everything is going well
+ * @param {object} requests - MojaloopRequests instance
```

Comment:
jsdoc doesn't match the function's params

Reply:
fixed!